timesketch/api/v1/routes.py

```diff
@@ -104,6 +105,7 @@
     (VersionResource, '/version/'),
     (SigmaListResource, '/sigma/'),
     (SigmaResource, '/sigma/<string:rule_uuid>/'),
```

Comment:
Based on discussion, I will deprecate this and replace with:

```
sigma/text/
```
and 
```
sigma/rule/<string:rule_uuid>
```

Reply:
yes

Reply:
implemented